src/VoxaApp.ts

```diff
@@ -468,23 +468,29 @@ export class VoxaApp {
     if (!transition.to) {
       throw new Error("Missing transition");
     }
-
-    if (typeof transition.to === "string"  ) {
-      voxaEvent.model.state = transition.to;
-    } else if (isState(transition.to)) {
-      voxaEvent.model.state = transition.to.name;
-    }
-
-    await response.saveSession(voxaEvent);
+    const stateName = typeof transition.to === "string" ? transition.to
+    : isState(transition.to) ? transition.to.name
+    : "";
+    if (!stateName) { throw new Error("Expected transition to transition to something"); }
```

Comment:
let's add a custom error type to https://github.com/mediarain/voxa/tree/v3/src/errors

Reply:
Added